cmd/dispenser/server.go

```diff
@@ -71,14 +71,24 @@ const topPageTemplate = `
     <script
       src="https://code.jquery.com/jquery-3.3.1.min.js"
       integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
-      crossorigin="anonymous"></script>
-
-    <script>
+	   crossorigin="anonymous"></script>
+	  
+	 <script>
+	  function loadparm(){
+			const queryString = window.location.search;
+			const urlParams = new URLSearchParams(queryString);
+			const parmTarget = urlParams.get('account');
```

Comment:
you need to check if urlParams.has('account'), since if it wasn't passed on, you don't want to attempt and assign the value.

Reply:
If it was not passed in, the value assigned will be null